feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -24,61 +24,119 @@ <h2>{{ 'configurator.overviewForm.noAttributeHeader' | cxTranslate }}</h2>
-          </ng-container>
+        <span class="cx-visually-hidden">
+          {{
+            'configurator.a11y.group'
+              | cxTranslate
+                : {
+                    group: group.groupDescription
+                  }
+          }}
+        </span>
+        <h2 aria-hidden="true">
+          <span>{{ group.groupDescription }}</span>
+        </h2>
+
+        <div
+          *ngFor="let attributeOverview of group.attributes; let i = index"
+          class="cx-attribute-value-pair"
+          [ngClass]="getStyleClasses(group.attributes, i)"
+        >
+          <ng-container [ngSwitch]="attributeOverview?.type">
+            <ng-container *ngSwitchCase="attributeOverviewType.GENERAL">
+              <cx-configurator-overview-attribute
+                [attributeOverview]="attributeOverview"
+              >
+              </cx-configurator-overview-attribute>
+            </ng-container>
 
-          <ng-container *ngSwitchCase="attributeOverviewType.BUNDLE">
-            <cx-configurator-cpq-overview-attribute
-              [attributeOverview]="attributeOverview"
-            >
-            </cx-configurator-cpq-overview-attribute>
+            <ng-container *ngSwitchCase="attributeOverviewType.BUNDLE">
+              <cx-configurator-cpq-overview-attribute
+                [attributeOverview]="attributeOverview"
+              >
+              </cx-configurator-cpq-overview-attribute>
+            </ng-container>
+
+            <ng-container *ngSwitchDefault>
+              <cx-configurator-overview-attribute
+                [attributeOverview]="attributeOverview"
+              >
+              </cx-configurator-overview-attribute>
+            </ng-container>
           </ng-container>
+        </div>
+      </div>
+      <ng-container *ngIf="group.subGroups?.length > 0">
+        <ng-container
+          *ngTemplateOutlet="
+            groups;
+            context: { overviewGroups: group.subGroups, level: level + 1 }
+          "
+        ></ng-container>
+      </ng-container>
+    </ng-container>
+  </ng-container>
+
+  <ng-container *cxFeatureLevel="'6.0'">
+    <ng-container *ngFor="let group of overviewGroups">
+      <div [ngClass]="getGroupLevelStyleClasses(level, group.subGroups)">
+        <span class="cx-visually-hidden">
+          {{
+            'configurator.a11y.group'
+              | cxTranslate
+                : {
+                    group: group.groupDescription
+                  }
+          }}
+        </span>
+        <h2 aria-hidden="true">
+          <span>{{ group.groupDescription }}</span>
+        </h2>
+
+        <div
+          *ngFor="let attributeOverview of group.attributes; let i = index"
+          class="cx-attribute-value-pair"
+          [ngClass]="getStyleClasses(group.attributes, i)"
+        >
+          <ng-container [ngSwitch]="attributeOverview?.type">
+            <ng-container *ngSwitchCase="attributeOverviewType.GENERAL">
```

Comment:
This switch-case, `attributeOverviewType.GENERAL`, is equal to default.
It is possible to simplify it, namely to use `*ngSwitchCase="attributeOverviewType.BUNDLE"` case and default?